tuna/example/example_tables.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+###############################################################################
+#
+# MIT License
+#
+# Copyright (c) 2023 Advanced Micro Devices, Inc.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+###############################################################################
+""" Module for creating DB tables"""
+import enum
+from sqlalchemy import Column, Integer, String, ForeignKey
+from sqlalchemy import Text, Enum
+from sqlalchemy.ext.declarative import declared_attr
+
+from tuna.dbBase.base_class import BASE
+from tuna.machine import Machine
+from tuna.example.session import SessionExample
+
+#pylint: disable=too-few-public-methods
+
+
+class JobEnum(enum.Enum):
+  """Represents job_enum column in config table"""
+  # pylint: disable=invalid-name ; names represent entries in job_enum column
+  # pylint: disable=duplicate-code
+  new = 1
+  started = 2
+  running = 3
+  completed = 4
+  timeout = 5
+  error_status = 6
+  no_update = 7
+  errored = 8
+  bad_param = 9
+  error = 10
+  transfer_error = 11
+  not_applicable = 20
+  aborted = 21
```

Comment:
@cderb this might be overkill, should be simplify?

Reply:
You can remove 7 and up, new teams will likely bring their own error statuses. 1-6 should be sufficient as an example.